tests/test_flows.py

```diff
@@ -2695,6 +2696,38 @@ def dog():
             load_flow_from_entrypoint(f"{fpath}:dog", use_placeholder_flow=False)
 
 
+class TestLoadFunctionAndConvertToFlow:
```

Comment:
Can you add a test covering when the function isn't found?

Reply:
Yes can do!